packages/redux-subspace-observable/src/observable/subspaceEpicMiddleware.js

```diff
@@ -0,0 +1,21 @@
+import { applyToRoot } from 'redux-subspace'
+import { createEpicMiddleware as baseCreateEpicMiddleware } from 'redux-observable'
+import { SUBSPACE_STORE_KEY } from './subspaceStoreKey';
+
+export const createEpicMiddleware = (rootEpic, options = {}) => {
+    if (process.env.NODE_ENV !== 'production') {
+        const dependenciesIsObject = options.dependencies == undefined || typeof options.dependencies === 'object'
+        console.assert(dependenciesIsObject, 'dependencies must be an object')

```

Comment:
How about an exception, or a `console.warn`? `console.assert` has inconsistent behavior across browsers and node--in some it throws, in others it just logs an error but doesn't throw. Node is used when people server-side render. Personally I would throw, like you did in https://github.com/ioof-holdings/redux-subspace/pull/66/files#diff-d6110079ec3df7516f551a2c9957fad6R20

Reply:
The things you're teaching me @jayphelps.... we've used `console.assert` as the default way to raise errors within redux-subspace.  The intention was to throw, so if `assert` is not consistent at doing that, we should change them all.

I did wonder why other libraries seemed not to use it as it seemed perfect for what I wanted.

Reply:
Lets fix them all in another PR.

Reply:
Agreed. Well beyond the scope of this change.